src/distributed/wallet.did

```diff
@@ -7,6 +7,7 @@ type EventKind = variant {
   CyclesReceived: record {
     from: principal;
     amount: nat64;
+    memo: opt text;
```

Comment:
Please add this .did file and the new wallet.wasm here: https://github.com/dfinity/sdk/tree/master/e2e/archive/wallet